content/npm-public-beta.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "Sigstore Support in NPM launches for Public Beta"
+date = "2023-02-23"
```

Comment:
Should this date be swapped out for today?

Reply:
Whoops, very much